index.html

```diff
@@ -1,12 +1,270 @@
 <html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Девайс</title>
-  </head>
-  <body>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>HTML Academy: Девайс</title>
+</head>
+
+<body>
+
+<header class="site-header">
+
+  <nav class="user-navigation">
+    <img class="logo" src="img\logo.svg" alt="Логотип Device">
+    <h1 class="visually-hidden">Device</h1>
+    <form class="user-navigation search-form" action="https://echo.htmlacademy.ru/" method="POST">
+      <div class="user-navigation search-item">
+        <img class="user-navigation search-item seacrh-ico" alt="Поиск">
+        <input class="user-navigation search-item seacrh-input" type="search" placeholder="Текст поиска">
+      </div>
+      <input class="user-navigation search-item submit-item" type="submit" value="Найти">
+    </form>
+
+    <img class="user-navigation sign-in-ico" src="img\account.svg" alt="Войти">
+    <a href="#" class="user-navigation sign-in">Войти</a>
+    <ul class="user-navigation-list">
+      <li class="user-navigation-item">
+        <img class="user-navigation-link user-navigation-ico" src="img\compare.svg" alt="Сравнить">
+        <a class="user-navigation-link" href="#">Сравнить</a>
+      </li>
+      <li class="user-navigation-item">
+        <img class="user-navigation-link user-navigation-ico cart-counter-ico" src="img\cart_count.svg" alt="Кол-во заказов в корзине">
+        <span class="user-navigation-link user-navigation-ico cart-count">2</span>
+        <img class="user-navigation-link user-navigation-ico cart-ico" src="img\cart.svg" alt="Корзина">
+        <a class="user-navigation-link cart" href="#">Корзина</a>
+      </li>
+    </ul>
+  </nav>
+
+  <nav class="main-navigation">
+    <ul class="main-navigation-list">
+      <li class="main-navigation-item">
+        <a class="main-navigation-link" href="catalog.html">Каталог товаров</a>
+        <img class="main-navigation-link main-navigation-ico" src="img\catalog-toggler.svg" alt="Каталог">
+      </li>
+      <li class="main-navigation-item">
+        <a class="main-navigation-link" href="#">Доставка</a>
+      </li>
+      <li class="main-navigation-item">
+        <a class="main-navigation-link" href="#">Гарантия</a>
+      </li>
+      <li class="main-navigation-item">
+        <a class="main-navigation-link" href="#">Контакты</a>
+      </li>
+    </ul>
+    <div class="main-navigation-background"></div>
+  </nav>
+
+  <aside class="product-preview">
+
+    <article class="product-preview-constructor">
+      <img class="product-preview-img" src="img/quadrocopter_with_laser.png">
+      <p class="product-preview-index">01</p>
+      <b class="product-preview-slogan">Порхает как бабочка,<br>жалит как пчела!</b>
```

Comment:
```suggestion
      <p class="product-preview-slogan">Порхает как бабочка,<br>жалит как пчела!</p>
```

не надо использовать презентационные теги в html, чтобы разделять сущности: html отвечает за разметку, а за визуал и стили отвечает css

Reply:
Уже убрал

Reply:
Разные теги нужно использовать, потому что в них как раз и заложен смысл, они вообще-то не просто какие-то левые буквы: они сокращения от английских слов, которые обозначают сущность, для которой эти теги сделаны! Верстать разными тегам - это верстать семантично. Ты суп ешь ложкой, а второе - вилкой, а режешь ножом. Ты выучил как пользоваться разными приборами для разной еды, значит и теги выучишь